### ISSUE TITLE
Geofence Enter notification issue when setting new geofence

### ISSUE THREAD

**Issue by victor-butler** [traccar/traccar#5218]
**Describe the bug**
Every time when a geofence is created with the device **inside** the geofence zone, the **Geofence Enter** notification is triggered upon receiving the next data packet.
This is not correct because if the vehicle is parked and the user creates a geofence around it, the next record that is sent triggers a **Geofence Enter**, when in fact the vehicle hasn't moved.

**To Reproduce**
1. Create a geofence with the vehicle inside the geofence area
2. Wait for next data record to arrive
3. Geofence Enter event is created (this is wrong because if the record arrives 2 hours after creating the geofence and the vehicle hasn't moved, the user will be confused).

**Expected behavior**
The Geofence Enter event should be trigged only under two conditions (like it was before version 5.7):

- Geofence is created when the device is outside the geofence -> Event is triggered when the device enters geofence
- Geofence is created when the device is inside the geofence ->  Event is created ONLY after the device exists geofence AND enters the geofence again.

**Desktop (please complete the following information):**
  - All browsers
 - Version 5.7 onwards

**Comment by tananaev** [comment]
I believe we have already discussed it previously. I disagree that it's a bug.

**Comment by victor-butler** [comment]
Yes, we've discussed it. 

No matter how you call it, it doesn't work well if the user receives a Geofence Enter notification, 2, 3, 4 or 5 hours after the geofence is created but and the vehicle didn't move at all.

Please, reconsider.
